src/job/jobHandler.ts

```diff
@@ -234,21 +234,90 @@ export abstract class JobHandler {
     }
   }
 
+  private async loggingMessage(resp: CommandExecutorResponse): Promise<void> {
```

Comment:
minor. would rename to `logBuildDetails`

Reply:
I will make the change to the method name.